extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventActionExecutionFieldsMixin.schema.json

```diff
@@ -222,6 +222,63 @@
                   }
                 }
               }
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/actionOriginEndpoint": {
+              "title": "actionOriginEndpoint",
+              "description": "Full Endpoint of the Custom Action Used",
+              "type": "string",
+              "meta:titleId": "journeyStepEventActionExecutionFields##https://ns.adobe.com/experience/journeyOrchestration/actionOriginEndpoint##title##34643",
```

Comment:
You can drop all meta:titleId and meta:descriptionId fields; they are autogenerated